README.md

```diff
@@ -35,11 +35,25 @@ Fallbacks for older browsers are the responsibility of the developer adding Hove
 
 Please be certain any effects you submit at least work in the latest versions of modern browsers, and advise us accordingly if that is not the case.
 
-# To Do List
 
-The following is a list of ideas we have in mind for developing incuna-transitions:
+# Working on the project
 
-1. Remove all trace of icon-fonts. If you would like to learn more about why icon-fonts are bad you can watch [this video](https://www.youtube.com/watch?v=9xXBYcWgCHA) by our excellent contributor @ninjanails.
-2. Add the option to introduce arguments to mixins to make them more customisable
-3. Create some really great page transitions and other effects that will pair nicely with Angular JS's functionality.
-4. And more...
+This project uses `grunt` to compile sass and templates. 
+
+To set up the project locally:
+* Run `npm install`
+* Run `bower install`
```

Comment:
I don't think that's necessary. Only for large projects that have required install steps that aren't obvious based on the files: `package.json` and `bower.json` both tell the dev to `npm install` and `bower install`, respectively.